.github/workflows/visual.yml

```
@@ -0,0 +1,139 @@
+      contents: read
+    if: github.event.workflow_run.head_repository.full_name == github.repository
+    uses: ./.github/workflows/ignore-stale-commits.yml
+    with:
+      head_repository: ${{ github.event.workflow_run.head_repository.full_name }}
+      head_branch: ${{ github.event.workflow_run.head_branch }}
+      head_sha: ${{ github.event.workflow_run.head_sha }}
+
+  update-snapshots:
+    name: Update snapshots
+    needs: gate
+    if: needs.gate.outputs.is_latest == 'true' && env.USE_VIZZLY != 'true' && github.event.workflow_run.conclusion == 'failure'
```

actionlint (1.7.11)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)